modules/admission-webhook-create.adoc

```diff
@@ -0,0 +1,375 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-controllers.adoc
+
+[id="admission-webhook-create_{context}"]
+= Creating an admission webhook
+
+This procedure outlines high level steps to deploy a webhook server as a DaemonSet and to deploy the webhook server as an aggregated API server. Deploying the webhook server as an aggregated API server allows other {product-title} components to communicate with the webhook using internal credentials and facilitates testing using the `oc` command. This also enables role based access control (RBAC) into the webhook and prevents token information from other API servers being disclosed to the webhook. The API server calls `kube-apiserver` with its fully authenticated service account. The aggregated API server then calls to the webhook using the request header certificate. The webhook verifies incoming client certificates through the published request header aggregation CA.
```

Comment:
* high level -> high-level
* I think you need another "from" in front of "being disclosed". wdyt?
* Should `kube-apiserver` be "the Kubernetes API server"?

Reply:
I agree, I think "from being disclosed" is better.

All three items now resolved.